graphql/resolvers.go

```diff
@@ -187,6 +187,22 @@ func (r *patchResolver) CommitQueuePosition(ctx context.Context, apiPatch *restM
 	return commitQueuePosition, nil
 }
 
+func (r *patchResolver) TaskStatuses(ctx context.Context, obj *restModel.APIPatch) ([]string, error) {
+	tasks, _, err := r.sc.FindTasksByVersion(*obj.Id, task.DisplayNameKey, []string{}, "", "", 1, 0, 0)
```

Comment:
If this query is going to be polling periodically, can you either add a parameter to this function to project just the variant and status, or use an aggregation to return statuses grouped by variant? If not disregard

Reply:
great suggestion. done!